AUTHORS

```diff
@@ -181,6 +182,7 @@ Li Zhilin
 Ling Wang
 Linus Dietz
 Lorenzo Genta
+Lucas Beretti
```

Comment:
There should be no need to update the file. It is automatically generated by https://github.com/JabRef/jabref/blob/master/scripts/generate-authors.sh

I hope, you used `Co-authored-by:` (https://help.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors) at your commits. Otherwise, the names could disappear (without having generate-authors.sh modified).

There are thoughts of removing AUTHORS, because maintenance is hard. That said, it is even more important to have used `Co-authred-by:`